doc/getting-started/building-from-source.rst

```diff
@@ -3,6 +3,46 @@
 Build ACRN from Source
 ######################
 
+Introduction
+************
+
+Following general embedded system programming model, ACRN hypervisor
```

Comment:
Following general embedded system programming model,  ACRN hypervisor is designed to be customized per hardware platform and per usage scenario, rather than one binary for all platforms/all usage scenarios.